python/ray/_private/runtime_env/conda_utils.py

```diff
@@ -194,6 +194,23 @@ def get_conda_info_json() -> dict:
     return json.loads(stdout)
 
 
+def validate_ray_installed_in_conda_env(env: str):
+    """
+    Get `conda list -n env --json`
+
+    Check if ray is installed in the conda env.
+    """
+    conda_path = get_conda_bin_executable("conda")
+    try:
+        exec_cmd([conda_path, "--help"], throw_on_error=False)
+    except EnvironmentError:
+        raise ValueError(f"Could not find Conda executable at {conda_path}.")
+    _, stdout, _ = exec_cmd([conda_path, "list", "-n", env, "ray", "--json"])
```

Comment:
I tried running this command in my local environment. It seems to list all packages that contain the string "ray." Hence, it may produce false positives.

<img width="643" alt="image" src="https://github.com/user-attachments/assets/6dbee3fb-7b58-4542-a54a-1fcea02cb7fa" />

This function may need to iterate over all matching packages and ensure that the ray package exists.

<img width="798" alt="image" src="https://github.com/user-attachments/assets/5a94fba2-6c65-4ba4-a4b1-713fb0306fff" />